clang/lib/Headers/intrin.h

```diff
@@ -329,6 +329,25 @@ static __inline__ void __DEFAULT_FN_ATTRS __stosq(unsigned __int64 *__dst,
 static __inline__ void __DEFAULT_FN_ATTRS __halt(void) {
   __asm__ volatile("hlt");
 }
+
+static inline int _inp(unsigned short port) {
+  int ret;
+  __asm__ volatile("inb %w1, %b0" : "=a"(ret) : "Nd"(port));
```

Comment:
I took a look at the Clang code, it seems these `b`, `w` modifiers are only meaningful to ARM/AArch64. We don't need them for X86 assembly.

Reply:
I have a check, they are still required.